### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Description of why you made this PR*

## Solution
- *And how do you fix that problem*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
*If you have access to [App Continuous Integration for packagers](https://yunohost.org/#/packaging_apps_ci) you can provide a link to the package_check results like below, replacing '-NUM-' in this link by the PR number and USERNAME by your username on the ci-apps-dev. Or you provide a screenshot or a pastebin of the results*

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/opensondage_ynh%20PR-NUM-%20(USERNAME)/)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Review by alexAubin — APPROVED** [review]
LGTM

File: scripts/_common.sh [hunk 1]
```diff
@@ -1,5 +1,22 @@
 #!/bin/bash
 
+#=================================================
+# COMMON VARIABLES
+#=================================================
+
+# dependencies used by the app
+YNH_PHP_VERSION="7.3"
+
+pkg_dependencies="php${YNH_PHP_VERSION}-fpdf php${YNH_PHP_VERSION}-cli php${YNH_PHP_VERSION}-xml"
```
Comment (alexAubin) [inline scripts/_common.sh:10]: Apparently tho the CI complains that : 

` E: Package 'php7.3-fpdf' has no installation candidate`

**Review by alexAubin — CHANGES_REQUESTED** [review]

**Comment by yalh76** [comment]
!testme

**Pull request merged**